docs/reference/admin/layouts/slots.mdx

```diff
@@ -3,51 +3,59 @@ title: Slots
 ---
 
 import CodeBlock from '@theme/CodeBlock'
-import _components_SlotsSource from '!!raw-loader!./_Example/components/Slots.ts'
+import _components_SlotsSource from '!!raw-loader!./_Example/components/Slots.tsx'
 import _pages_example_Source from '!!raw-loader!./_Example/pages/example.tsx'
 
 # <span className="version">Interface 1.2+</span> Slots
 
+**Use Slots to render contents of the current route to the other areas of the layout.**
+
+Every Contember component can [access data binding context](/reference/admin/data-binding/overview/)
+and render it to the page but this composition is limited to how the content might be rendered as is.
+
+For example you can't render a persist button to save the form in the header from the page because
+it is nested within the descendants of the page that is rendered in the layout.
+
+To overcome this limitation slots provide a way to *escape*  and render to other parts of the layout easily, even
+the parts that are not directly descendants of the page, e.g. header or footer of the layout.
+
+## 1. What are slots?
```

Comment:
pls do not use numbered sections, it only makes sense in soem step-by-step guides

Reply:
Done